DATA_MODEL.md

```
@@ -0,0 +1,135 @@
+  "original_html_snippet": "string | null (The original HTML/text snippet this ruling was derived from)",
+  "tags": ["string", "..."]
+}
+```
+
+**Fields:**
+
+*   `id`: A unique UUID string identifying the ruling.
+*   `source_card_code`: The 5-digit code of the card this ruling was primarily found under or is most associated with. "00000" can indicate a general ruling not tied to a specific card.
+*   `related_card_codes`: A list of other 5-digit card codes that this ruling pertains to or mentions.
+*   `ruling_type`: An enum string indicating the type of ruling. Possible values include:
+    *   `ERRATA`: Official correction to card text or game rules.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)